implicit 'any' return type

When I was compiling using TSC with the following configs, I see this errors.

```json
{
  "compilerOptions": {
    "sourceMap": true,
    "outDir": "dist",
    "strict": true,
    "lib": ["esnext"],
    "esModuleInterop": true,
    "module": "commonjs",
    "target": "ES6",
    "typeRoots": ["./node_modules/@types", "./src/typings"]
  }
}
```

```bash
src/typings/excel4node/index.d.ts(11,5): error TS7010: 'addDataValidation', which lacks return-type annotation, implicitly has an 'any' return type.

src/typings/excel4node/index.d.ts(16,5): error TS7010: 'addConditionalFormattingRule', which lacks return-type annotation, implicitly has an 'any' return type.

src/typings/excel4node/index.d.ts(46,5): error TS7010: 'addImage', which lacks return-type annotation, implicitly has an 'any' return type.

src/typings/excel4node/index.d.ts(96,19): error TS7010: 'getExcelAlpha', which lacks return-type annotation, implicitly has an 'any' return type.

src/typings/excel4node/index.d.ts(109,19): error TS7010: 'getExcelTS', which lacks return-type annotation, implicitly has an 'any' return type.

--- a/typings/excel4node.d.ts
+++ b/typings/excel4node.d.ts
@@ -8,12 +8,12 @@ declare module 'excel4node' {
     /**
      * Accepts a validation options object with these available options. All options are optional with exception of sqref.
      */
-    addDataValidation(opt: WorksheetValidation): any;
+    addDataValidation(opt: WorksheetValidation): void;
     /**
      * Conditional formatting adds custom formats in response to cell reference state. A subset of conditional formatting features is currently supported by excel4node.
      * Formatting rules apply at the worksheet level.
      */
-    addConditionalFormattingRule(ref: string, opt: AddConditionalFormattingRuleOptions): any;
+    addConditionalFormattingRule(ref: string, opt: AddConditionalFormattingRuleOptions): void;
 
     column(number: number): Column;
 
@@ -29,7 +29,7 @@ declare module 'excel4node' {
     /**
      * Adds and image to the worksheet.
      */
-    addImage(opt: AddImageOptions): any;
+    addImage(opt: AddImageOptions): void;
   }
   /**
    * An instance of the Workbook class contains all data and parameters for the Excel Workbook.

--- a/typings/excel4node.d.ts
+++ b/typings/excel4node.d.ts
@@ -8,12 +8,12 @@ declare module 'excel4node' {
     /**
      * Accepts a validation options object with these available options. All options are optional with exception of sqref.
      */
-    addDataValidation(opt: WorksheetValidation);
+    addDataValidation(opt: WorksheetValidation): any;
     /**
      * Conditional formatting adds custom formats in response to cell reference state. A subset of conditional formatting features is currently supported by excel4node.
      * Formatting rules apply at the worksheet level.
      */
-    addConditionalFormattingRule(ref: string, opt: AddConditionalFormattingRuleOptions);
+    addConditionalFormattingRule(ref: string, opt: AddConditionalFormattingRuleOptions): any;
 
     column(number: number): Column;
 
@@ -29,7 +29,7 @@ declare module 'excel4node' {
     /**
      * Adds and image to the worksheet.
      */
-    addImage(opt: AddImageOptions);
+    addImage(opt: AddImageOptions): any;
   }
   /**
    * An instance of the Workbook class contains all data and parameters for the Excel Workbook.
@@ -73,7 +73,7 @@ declare module 'excel4node' {
   /**
    * Accepts column as integer and returns corresponding column reference as alpha. (i.e. returns 'J')
    */
-  export function getExcelAlpha(column: number);
+  export function getExcelAlpha(column: number): string;
 
   /**
    * Accepts row and column as integers and returns Excel cell reference. (i.e. returns 'C5')
@@ -83,7 +83,7 @@ declare module 'excel4node' {
   /**
    * Accepts Date object and returns an Excel timestamp. (i.e. Returns 42004.791666666664)
    */
-  export function getExcelTS(date: Date);
+  export function getExcelTS(date: Date): number;
 
   interface WorkbookOption {
     jszip?: {

